docs/instrumenten/IAMA.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Impact Assessment Mensenrechten en Algoritmes
+toelichting: Het Impact Assessment voor Mensenrechten bij de inzet van Algoritmes (IAMA) is een instrument voor overheidsorganen om een interdisciplinaire dialoog en besluitvorming te faciliteren bij de ontwikkeling en inzet van algoritmische systemen. 
+vereiste:
+- beoordelen_gevolgen_voor_grondrechten
+levenscyclus:
+- ontwerp
```

Comment:
is het IAMA niet toepasbaar in meerdere fasen? Ik denk bijvoorbeeld ook aan verificatie en validatie? of zelfs implementatie?

Reply:
Ja volgens mij heb je helemaal gelijk! Die, en ook nog Probleemanalyse toegevoegd!